### PR TITLE
fix(): the directive shouldn’t validate empty values

### PR DESCRIPTION
**Pull request by gmq**
Simpler than #14, there's no need to get `required` attribute since Angular should be the one managing that.

### CODE REVIEW

**Comment by blackjid** [comment]
@gmq i'm with you that the `required` should be managed by angular itself. On the other hand, what is the relation between this PR an #14? without this fix the `required` doens't work??

what do you think about adding a test to cover the use of `required` and the `empty` validation?

**Comment by gmq** [comment]
@blackjid

Without this PR the rut field is _always_ required since an empty value fails the rut validation, so if you check the form's validity before submission it'll fail if the rut field is empty even if it doesn't have required attribute.

 #14 solves it by letting empty values go through but it checks for the required attribute before doing it which I think is a bit unnecessary.

I'll add a test ASAP.

File: test/rut_spec.js [hunk 1]
```diff
@@ -50,6 +50,11 @@ describe('', function() {
       expect(scope.form.rut.$valid).toEqual(false);
     });
 
+    it('should pass with an empty value', function() {
+      setInputValue(element, '');
+      expect(scope.form.rut.$valid).toEqual(true);
```
Comment (monkeyci) [inline test/rut_spec.js]: 'expect' is not defined.

**Comment by blackjid** [comment]
Thanks!, What do you think we should do with the annoying monkey's comments???

also, where do we have an ongoing conversation about whether we should commit the built code or no??

**Comment by gmq** [comment]
I feel both issues are outside the scope of this PR 🤔

I'm not sure about removing the dist folder from bower-managed libraries, bower's registry uses github, unlike npm which has its own thing going on.

**Comment by blackjid** [comment]
mm, yes, probably I'd like it better to have another repo, like `bower-angular-rut` with the distribution code.... but, I also find it a little overkill for small projects, unless we have an automatic way to do it, it would be too much of a hassle to maintain.

i'll go ahead and merge this. Later we could talk about bower, repo, ci, etc

**Comment by jaimeguaman** [comment]
Thanks!!!!!
